src/components/MessageList/MessageList.jsx

```diff
@@ -45,15 +40,65 @@ class MessageList extends Component {
     }
   }
 
-  render () {
-    const { messages, typing, activeId } = this.props;
+  renderTyping () {
+    // TODO: Make this somehow not a giant eyesore on this file
```

Comment:
👍 

I added a similar comment in the client -- we'll get around to it... maybe a shared library folder for things like CSS (message styles), functions and components ...